.github/workflows/e2e-pw-pull-request.yml

```diff
@@ -6,6 +6,9 @@ on:
             - develop
             - trunk
     workflow_dispatch:
+    pull_request_review:
+        types:
+            - submitted
```

Comment:
What would happen if E2E tests are run as usual and then someone adds a new review? Would the tests trigger again?

Reply:
Yes, you're right, `pull_request.ready_for_review` action seems like the correct event that should trigger this.